backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/CloneImageGroupVolumesStructureCommand.java

```diff
@@ -184,8 +184,26 @@ private void createImage(DiskImage image, int imageIndex) {
         runInternalActionWithTasksContext(ActionType.CreateVolumeContainer, parameters);
     }
 
+    /**
+     * Below is the table that describes what the method does.
+     * Note: for Live Storage Migration: volumeFormat = source volume format == destination volume format.
+     *
+     *
+     * Type     Source Domain   Destination Domain   Volume Format       Initial Size (method's return value)
+     * =================================================================================================================
+     * Active       file              file              qcow2                         null
+     * Active       file              block             qcow2         3 * VolumeUtilizationChunkInMB
+     * Active       block             block             qcow2         3 * VolumeUtilizationChunkInMB
+     * -----------------------------------------------------------------------------------------------------------------
+     * Internal     file               file              raw                          null
+     * Internal     file               file              qcow2                        null
+     * Internal     file               block             raw                          null
+     * Internal     file               block             qcow2             MeasureVolumeCommand return value
+     * Internal     block              block             raw                          null
+     * Internal     block              block             qcow2             MeasureVolumeCommand return value
```

Comment:
Values should be aligned to the left, including the titles.
Also use shorter titles to this is not too wide.

I would use something like:

| type     | src type | dst type | vol format | initial size |
|----------|----------|----------|------------|--------------|
| active   | block    | block    | qcow2      | 3 chunks     |
| internal | block    | block    | qcow2      | measure      |

Reply:
Done.
1. Aligned to the left.
2. Used shorter titles and cells content.
3. Some other improvements.